lib/Conversion/XTenNNToTorch.cpp

```diff
@@ -149,6 +154,51 @@ ValueRange groupConv2dToTorch(GroupConv2dOp op, GroupConv2dOp::Adaptor adaptor,
       ->getResults();
 }
 
+ValueRange padReflectToTorch(ReflectPadOp op, ReflectPadOp::Adaptor adaptor,
+                              ArrayRef<Type> types, ValueRange values,
+                              ConversionPatternRewriter &rewriter) {
+  auto loc = op->getLoc();
+  auto opName = rewriter.getStringAttr("onnx.Pad");
```

Comment:
We are seeing a lot of this opName being created for all TorchONNX operations, which make sense. I'll approve the PR but we could also have a base class `ApplyXTenNNToTorchONNX` that inherents from `ApplyXTenNNToTorch` and takes care any attribute that is specific to all operations.

Reply:
Maybe for later?

Reply:
Yes, this could definitely be improved